public/index.php

```diff
@@ -16,5 +16,25 @@
 $channel->publish("play", "meow");
 $channel->publish("play", "cluck");
 
+echo "Releasing channel...\n";
```

Comment:
mostly to indicate that typical operation would end here, and to show that it's possible to free memory of unused channels. I can skip entire release/retrieve altogether, since as you said it's not necessary.